release.yaml

```diff
@@ -1,26 +1,4 @@
-fleet:
-- 100.1.0+up0.4.0
-fleet-agent:
-- 100.1.0+up0.4.0
-fleet-crd:
-- 100.1.0+up0.4.0
-rancher-eks-operator:
-  - 100.0.4+up1.1.5
-rancher-eks-operator-crd:
-  - 100.0.4+up1.1.5
-rancher-aks-operator:
-  - 100.0.6+up1.0.7
-rancher-aks-operator-crd:
-  - 100.0.6+up1.0.7
-rancher-csp-adapter:
-  - 1.0.1
-rancher-backup:
-- 2.1.4
-rancher-backup-crd:
-- 2.1.4
-rancher-webhook:
-  - 1.0.6+up0.2.7
-neuvector:
-- 100.0.2+up2.2.3
-neuvector-crd:
-- 100.0.2+up2.2.3
```

Comment:
Not sure why these are removed (by the script I think)...

Reply:
It need to be removed since we are only updating Longhorn in this PR.